### ISSUE TITLE
Remove hardcoded `python-version-support-pypi` badge

### ISSUE THREAD

**Issue by sugatoray** [SeldonIO/alibi-detect#441]
At present the badge used for showing supported python-version(s) is hard coded. There is a simpler alternative: use `python-version-support-pypi` badge that auto-updates the information based on PyPI release. This requires the use of classifiers in setup.py.

The package on PyPI does not show any python version. For clarity, please see the comparison between `doubtlab` and `alibi-detect` on supported python version(s) from PyPI.

<details>
<summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/10201242/151689401-bf7a4a83-cc19-49c2-804d-60ffa0f5a71c.png)

</details>

**For `doublab`**

<p>
  <a href="https://pypi.org/project/doubtlab/">
    <img src="https://img.shields.io/pypi/pyversions/doubtlab?logo=pypi&style=flat&color=blue" alt="PyPI - Supported Python Version" />
  </a>
</p>

**For `alibi-detect`**

<p>
  <a href="https://pypi.org/project/alibi-detect/">
    <img src="https://img.shields.io/pypi/pyversions/alibi-detect?logo=pypi&style=flat&color=blue" alt="PyPI - Supported Python Version" />
  </a>
</p>

## Suggested Fix

To fix this you need something like this. Please decide on the list of classifiers (see below) that you would like to use.

```python
# file: setup.py

setup(
	...
	...
	...
    classifiers=[
        "Intended Audience :: Science/Research",
        'Operating System :: OS Independent',
        "Programming Language :: Python :: 3",
        "Programming Language :: Python :: 3.7",
        "Programming Language :: Python :: 3.8",
        "Programming Language :: Python :: 3.9",
        "License :: OSI Approved :: Apache 2.0 License",
        "Topic :: Scientific/Engineering",
    ],
)
```

**Comment by sugatoray** [comment]
cc: @jklaise @ascillitoe

**Comment by ascillitoe** [comment]
These classifiers will be merged in #442, but I presume the python version badge will show *missing* until we release a new version on PyPI. Hence we should keep this issue open until the next release.

**Comment by sugatoray** [comment]
Yes, for now, until you push the changes made in setup.py to pypi, you will need to use the code for the badge used previously. 

If you can push a pre-release to pypi, that will also work. However, if anyone tries to install the package, then the latest release (not prerelease) will be downloaded.

**Comment by ascillitoe** [comment]
Thanks @sugatoray!

**Comment by ascillitoe** [comment]
Moving this to the `0.9.1` milestone so it will pick up the `setup.py` classifiers released in `0.9.0`.
